plugins/EthanLog/source/EthanLogClient.cpp

```diff
@@ -312,8 +312,8 @@ bool EthanLogClient::shouldDrop()
 
         char messageBuf[128];
         snprintf(messageBuf, sizeof(messageBuf),
-                 "MESSAGE=Dropped %u log messages in last %lld seconds (most "
-                 "recently, %lld seconds ago) due to excessive rate",
+                 "MESSAGE=Dropped %u log messages in last %ld seconds (most "
+                 "recently, %ld seconds ago) due to excessive rate",
```

Comment:
I believe we should switch to `PRId64` instead of `%lld` here. The `%lld` -> `%ld` change system dependent, and on 32 bit systems we'd get this same warning again but the other way around.

Refer to:

[https://linux.m2osw.com/c-printf3-int-long-and-long-long-works-every-time](https://linux.m2osw.com/c-printf3-int-long-and-long-long-works-every-time)

Reply:
Fixed